Authenticator/Source/AppViewController.swift

```diff
@@ -92,8 +92,8 @@ extension AppViewController: AppPresenter {
 
         case .EntryForm(let formViewModel):
             if case .EntryForm = currentViewModel.modal,
-                let editController = modalNavController?.topViewController as? TokenFormViewController {
-                    editController.updateWithViewModel(formViewModel)
+                let entryController = modalNavController?.topViewController as? TokenFormViewController<TokenEntryForm> {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 121 characters (line_length)